collectors/python.d.plugin/alarms/alarms.chart.py

```diff
@@ -57,6 +58,9 @@ def _get_data(self):
 
         raw_data = loads(raw_data)
         alarms = raw_data.get('alarms', {})
+        if self.alarm_contains_words != '':
+            alarms = {alarm_name: alarms[alarm_name] for alarm_name in alarms for alarm_contains_word in
+                      self.alarm_contains_words.split(',') if alarm_contains_word in alarm_name}
```

Comment:
Perhaps to make a set in init() and just "x in y" here?

Reply:
More short version (requires split self.alarm_contains_words in init()):

```py
{name: value for name, value in alarms.items() if any(w in name for w in self.alarm_contains_words)}
```

Reply:
Will do

Reply:
i actually tried this first, doing the split() in __init__ and was not working for some reason.

Reply:
will have another go as would be better to do it all in init